### ISSUE TITLE
Another NCMC bug

### ISSUE THREAD

**Issue by jchodera** [choderalab/perses#17]
[This section](https://github.com/pgrinaway/perses/blob/run_example/perses/annihilation/ncmc_switching.py#L6-L12) uses the wrong context parameter names:
```python
default_functions = {
    'alchemical_sterics' : 'lambda',
    'alchemical_electrostatics' : 'lambda',
    'alchemical_bonds' : 'lambda',
    'alchemical_angles' : 'lambda',
    'alchemical_torsions' : 'lambda'
    }
```
It should be `lambda_[name]`, rather than `alchemical_[name]`, though we should probably rename this in `alchemy` release 2.0.

**Comment by pgrinaway** [comment]
Good catch! Fixing...

**Comment by jchodera** [comment]
Thanks!

**Comment by jchodera** [comment]
Fixed by #14 and #18.
